### ISSUE TITLE
Tuples can't be constants

### ISSUE THREAD

**Issue by fubuloubu** [vyperlang/vyper#1431]
### Version Information
* vyper Version (output of `vyper --version`): 0.1.0b9
* OS: linux
* Python Version (output of `python --version`): Python 3.6.6
* Environment (output of `pip freeze`):
```
flake8==3.5.0
mccabe==0.6.1
pycodestyle==2.3.1
pycryptodome==3.6.6
pyflakes==1.6.0
```

### What's your issue about?
Vyper seems to think a tuple constant is a struct
```python
FOO: constant((uint256, uint256)) = (1, 2)
# vyper.exceptions.StructureException: Incorrectly formatted struct
```

Also, when I try to do a memory variable, I get an unhandled exception:
```python
@public
def bar():
    foo: (uint256, uint256) = (1, 2)
# AttributeError: 'Tuple' object has no attribute 'id'
```

I think these are related potentially.

### How can it be fixed?
It seems like the compiler doesn't know how to deal with tuple types as memory/constant declarations.

**Comment by jacqueswww** [comment]
This is not a problem with `constant` but just a fact that we don't support tuples. I think it's better not supporting tuples as this introduces a bunch of not so readable/unfamiliar parts of python ala `a, b = (1, 2)`

**Comment by fubuloubu** [comment]
I actually really like that syntax haha.

If this is the case, the resolution here should be to make it clearer that Tuples are not supported up front.

**Comment by jacqueswww** [comment]
I like it too! But I do think it brings readability issues, especially to readers not familiar with python.

**Comment by fubuloubu** [comment]
True. So, that basically relegates tuples only to returns of externally called functions? How does one do the following:
```python
def foo() -> (uint256, address):
    return self.balance, self.address

def bar():
    bal, addr = self.foo()
```

**Comment by fubuloubu** [comment]
This is not an issue

**Comment by fubuloubu** [comment]
Whoops, the resolution here is to make it clearer in the error message that tuples aren't a supported datatype

**Comment by iamdefinitelyahuman** [comment]
Closed in #2029
